pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
Ouch, why would they do that?... Anyway, if we can assume (hopefully) that the game would fallback to file's default path if it's not found in its configuration file location, we can try to fixup those paths before the game starts. Coincidentally I wanted to try XSLT in action so I went ahead and implemented this in https://github.com/abbradar/nixpkgs/commits/freeorion -- feel free to cherry-pick if you think this is useful.